x/axelarnet/keeper/migrate_test.go

```diff
@@ -44,9 +47,18 @@ func TestMigrate6to7(t *testing.T) {
 				return nil
 			},
 		}
+		nexusModuleAccAddr = rand.AccAddr()
+		axelarnetModuleAccAddr = rand.AccAddr()
 		account = &mock.AccountKeeperMock{
-			GetModuleAddressFunc: func(_ string) sdk.AccAddress {
-				return rand.AccAddr()
+			GetModuleAddressFunc: func(module string) sdk.AccAddress {
```

Comment:
we should consider creating integration tests, especially for migrations, so we don't end up mocking out the relevant logic

Reply:
lol that's what I was talking about.

Reply:
sure, i thought you meant to still make it into an e2e test